index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>@Sir_Malcom</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <section class="parent1">
+      <nav class="dcp1 navbar">
+          <img class="logo" src="./elements/lg.png" alt="Logo">
+          <div></div>
+          <img class="menu" src="./elements/Enabled.jpg" alt="Humbugger menu">
+      </nav>
+      <div class="space"></div>
+      <article class="dcp1 art1">
+          <h4>
+        I’m Sir_Malcom
+        Glad to see you!
+          </h4>
+      </article>
+      <article class="dcp1 art2">
```

Comment:
- Please change the `<article>` tag to a `<p>` tag so as to stick with the proper HTML semantics